frame/staking/src/lib.rs

```diff
@@ -3595,6 +3753,12 @@ where
 				continue
 			}
 
+			// if this is a non-zero slash, schedule tasks to chill their nominations.
+			if !slash_fraction.is_zero() {
+				let task = SlashTask::new(stash.clone());
+				<SlashTaskExecutor<T>>::mutate(|e| e.add_task(task));
```

Comment:
The `add_task` complexity is linear to the amount of already existing tasks in storage because of decoding and encoding of all tasks by mutate. Are the number of queued slashes capped and benchmarked for the worst case?

Reply:
Agreed.